src/lib/validations/is-multi-site.ts

```diff
@@ -55,19 +57,20 @@ export async function isMultiSiteInSiteMeta( appId: number, envId: number ): Pro
 			error_type: 'GraphQL-MultiSite-Check-failed',
 			gql_err: GraphQlError,
 		} );
-		exit.withError( `StartImport call failed: ${ GraphQlError }` );
+		// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+		exit.withError( `StartImport call failed: ${ GraphQlError as Error }` );
 	}
 
-	if ( Array.isArray( res?.data?.app?.environments ) ) {
-		const environments = res.data.app.environments;
+	if ( Array.isArray( res.data.app?.environments ) ) {
+		const environments = res.data.app!.environments;
 		if ( ! environments.length ) {
-			isMultiSite.set( arguments, false );
+			isMultiSite.set( args, false );
 			return false;
 		}
 		// we asked for one result with one appId and one envId, so...
-		const thisEnv = environments[ 0 ];
-		if ( thisEnv.isMultiSite || thisEnv.isSubdirectoryMultisite ) {
```

Comment:
This one confuses me a bit. According to the schema, the field is named `isMultisite`, not `isMultiSite`.